types/jest/index.d.ts

```diff
@@ -37,6 +37,8 @@ declare var xtest: jest.It;
 declare const expect: jest.Expect;
 
 type ArgsType<T> = T extends (...args: infer A) => any ? A : never;
+type RejectedValue<T> = T extends PromiseLike<any> ? any : never;
```

Comment:
Yikes, these are global. I didn't catch that on ArgsType. We should be more careful with naming here. Can we switch to naming like JestArgsType and JestRejectedValue, or put them in the jest namespace? Consumers may have their own global type helpers like this with slightly different semantics.

Reply:
oh yeah. didn't know that was global. I'll update that

Reply:
done.